### ISSUE TITLE
[Web-Chrome] Accessibility Menu Icon Should Be Clearly Shown

### ISSUE THREAD

**Issue by devsreekrishna** [topcoder-platform/qa-fun#333]
1. Go To https://www.topcoder.com/

2. Login With Valid Credential Then See Right Side Bottom Accessibility Menu & Observe.

3. See Screenshot & Observe

4. Expected Result:- Accessibility Menu Icon Should Be Shown Clearly.

5. Actual Result:- Accessibility Menu Icon Is Not Showing Clearly.

6. Screenshot:-
[Accessibility .png.zip](https://github.com/topcoder-platform/qa-fun/files/4579494/Accessibility.png.zip)

7. Device:- Laptop

8. OS Version:- 
[Mac Version.png.zip](https://github.com/topcoder-platform/qa-fun/files/4579501/Mac.Version.png.zip)

9. Browser Information:-
[Chrome Version.png.zip](https://github.com/topcoder-platform/qa-fun/files/4579506/Chrome.Version.png.zip)

**Comment by winterflame** [comment]
cannot reproduce
<img width="539" alt="image" src="https://user-images.githubusercontent.com/4590317/81393387-dc719b00-9152-11ea-81cd-e050b96fa18f.png">
